eng/tools/lint-diff/src/lint-diff.ts

```diff
@@ -91,32 +88,32 @@ export async function main() {
   );
   console.log(`Using @microsoft.azure/openapi-validator version: ${validatorVersion}\n`);
 
-  await runLintDiff(
-    beforeArg as string,
-    afterArg as string,
-    changedFilesPath as string,
-    outFile as string,
-    baseBranch as string,
-    compareSha as string,
-    githubRepoPath as string,
-  );
+  const beforePath = beforeArg as string;
+  const afterPath = afterArg as string;
+
+  await runLintDiff(beforePath, afterPath, outFile as string, githubRepoPath as string);
 }
 
 async function runLintDiff(
   beforePath: string,
   afterPath: string,
-  changedFilesPath: string,
   outFile: string,
-  baseBranch: string,
-  compareSha: string,
   githubRepoPath: string,
 ) {
+  const baseSha = await getSha(resolve(beforePath));
+  const changedFiles = await getChangedFiles({
+    cwd: resolve(afterPath),
```

Comment:
```suggestion
    cwd: afterPath,
```

I don't think you **need** to resolve first.  It shouldn't hurt, but if you don't, the called API should do the right thing (resolve relative paths against cwd).  That said, I do find it best in our code, to resolve all path parameters as early as possible, and use the resolved versions in all subsequent code.  Example:

https://github.com/Azure/azure-rest-api-specs/blob/1ce249c1f9b04dd5f1883c82644cafa09e50c69c/.github/shared/src/spec-model.js#L39